beta/src/pages/learn/conditional-rendering.md

```diff
@@ -254,7 +255,7 @@ export default function PackingList() {
 
 </Sandpack>
 
-This style works well for simple conditions, but use it in moderation. If your components get messy with too much nested conditional markup, consider extracting child components to clean things up. In React, markup is a part of your code, so you can use tools like variables and functions to tidy up complex expressions.
+यह शैली साधारण परिस्थितियों के लिए अच्छी तरह से काम करती है, लेकिन इसे मॉडरेशन में उपयोग करें। यदि आपके कौम्पोनॅन्ट बहुत अधिक नेस्टेड कंडीशनल मार्कअप के साथ गड़बड़ हो जाते हैं, तो चीजों को साफ करने के लिए बाल कौम्पोनॅन्टस को निकालने पर विचार करें। React में, मार्कअप आपके कोड का एक हिस्सा है, इसलिए आप जटिल अभिव्यक्तियों को व्यवस्थित करने के लिए वेरिएबल और फ़ंक्शन जैसे टूल का उपयोग कर सकते हैं।
```

Comment:
`गड़बड़` => `कीच-पिच`

let me know how this sounds.

We can take a call on this.

Reply:
`कीच-पिच` sounds okay.